### ISSUE TITLE
Editor settings sidebar: Featured image missing grey placeholder box

### ISSUE THREAD

**Issue by autumnfjeld** [Automattic/wp-calypso#55566]
## The same issue reported in gutenberg https://github.com/WordPress/gutenberg/issues/30904

We should probably try to implement a core first solution (in which case we could close this ticket and work on the GB issue).  [Ian's suggestion](https://github.com/WordPress/gutenberg/issues/30904#issuecomment-839033426) is a nice one, though quite wordy to put all that text in the sidebar.  What about a pop-up guide?

### Steps to reproduce the behavior

1. Write a post (do not include a Featured image)
2. Click on publish
3. Scroll down to Featured Image in the side menu'
4. See that the "Set featured image" block is missing

THE EDITOR SETTINGS SIDEBAR :
![image](https://user-images.githubusercontent.com/5665959/129830104-aa77342d-19b5-4c88-8a27-a65de32d4afa.png)

THE SIDEBAR AFRER CLICKING PUBLISH
![image](https://user-images.githubusercontent.com/5665959/129830075-9b83c4b3-5ec0-4b10-b57e-767c0d0f29de.png?s=100)

#### What I expected to happen
The UI for the Featured Image section of the sidebar should be the same in the two scenarios pictured above.

###  SUGGESTION
Under the grey box, let's a support link to featured image docs:

["Learn about feature images"](https://wordpress.com/support/featured-images/)

This should match the style of other links in the sidebar. For example:
![image](https://user-images.githubusercontent.com/5665959/129830442-3953ac0d-1700-4bef-a2df-055c9bdd5a1b.png)

**Comment by arthur791004** [comment]
> The UI for the Featured Image section of the sidebar should be the same in the two scenarios pictured above.

Working this one on https://github.com/ampproject/amp-wp/pull/6563

**Comment by p-jackson** [comment]
There seems to be two problems described in this issue description:
1) The grey "set featured image" button is missing in the pre-publish panel
2) We should explain what a featured is and why it's important.

I think it makes sense to continue discussion about (2) in the GB repo WordPress/gutenberg#30904

(1) is fixed by ampproject/amp-wp#6563 which has been released in v2.1.4
The amp plugin on atomic sites has been updated to 2.1.4 because I think atomic updates automatically. However simple sites are still running 2.0.5. The last update to 2.0.5 looked pretty involved p58i-9pG-p2. We will need to kick off the process of getting the version updated on simple sites.

I'd say this is no longer a size small 😓

**Comment by arthur791004** [comment]
Yeah, I only deal with the first one. Oh, I don't know how to update the version for simple sites 😞 Learned and let me take a look for that posts.

**Comment by p-jackson** [comment]
ampproject/amp-wp#6563 has addressed issue (1), but for the fix to appear on WordPress.com we need to release the AMP plugin. See: pdgK6S-5M-p2

**Comment by autumnfjeld** [comment]
Work has been done on  D66398-code: Update AMP from 2.0.5 to 2.1.4,  but we'd like to pass this to the mobile web team, left a request here p1642575120002400-slack-C02DJJ0CLQN.
See more in P2 post: pdgK6S-5M-p2

**Comment by p-jackson** [comment]
Just to add my 2c: D66398-code is a proof of concept, feel free to progress this in whatever way makes sense.

**Comment by jordesign** [comment]
Hey @p-jackson  - keen for your gut check on this. I was thinking of closing it as (1) is resolved - and (2) is noted upstream. What do you think?

Any as we've moved on from AMP - any AMP specific issues seem super low priority?

**Comment by p-jackson** [comment]
@jordesign blast from the past!

Yeah I agree, with AMP not part of the dotcom experience anymore (pdKhl6-48-p2) this is fully an upstream issue now without anything dotcom specific.
